final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import logging
+import sys
+
+import feedparser
+
+from final_task.rss_reader.classes.exceptions import FeedError
+from final_task.rss_reader.classes.rss_feed import RssFeed
+from final_task.rss_reader.classes.rss_item import RssItem
+
+# common logger init
+logging.basicConfig(
+    stream=sys.stdout,
+    level=logging.DEBUG,
+    format='[%(asctime)s] {%(filename)s} %(levelname)s - %(message)s'
+    )
+logger = logging.getLogger('rss_logger')
```

Comment:
global variables usually declared in uppercase, e.g. `LOGGER`

Reply:
Done